ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -92,6 +96,7 @@ public CCDCase to(Claim claim) {
             .moreTimeRequested(claim.isMoreTimeRequested() ? YES : NO)
             .claimData(claimMapper.to(claim.getClaimData()))
             .defendantEmail(claim.getDefendantEmail())
+            .features(String.join(",", claim.getFeatures()))
```

Comment:
can `claim.getFeatures()`  be `null`? if yes, it will throw `NullPointerException` - If elements for join method is null